.github/scripts/commit-api-spec-change-log.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+echo "Running update changelog script"
+changeLog="$1"
+versionType="$2"
+echo "$changeLog"
+node .github/scripts/update-change-log.js "$changeLog"
+echo "Git configurations"
+git config --global user.email "team_interfaces+github@twilio.com"
```

Comment:
I tried to add email and name to secrets so that we can edit them without any code changes. But secrets are encrypted and not able to use them in this file. If we want to set it as environment variables we need to add something like this in the workflow file
```
run: |
          bash .github/scripts/commit-api-spec-change-log.sh "${{ github.event.inputs.change-log }}" ${{ github.event.inputs.version-type }}
env: 
  email: "team_interfaces+github@twilio.com"
  user: "twilio-dx"
```
it would same as adding them here. So didn't make any change.
@kridai If you have any option to set them as environment variable please let me know.